plugins/source/aws/resources/services/securityhub/findings.go

```diff
@@ -44,19 +46,46 @@ This is useful when multi region and account aggregation is enabled.`,
 
 func fetchFindings(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan<- any) error {
 	cl := meta.(*client.Client)
+	allConfigs := []tableoptions.CustomGetFindingsOpts{{}}
+	noTableConfig := true
+	if cl.Spec.TableOptions.SecurityHubFindings.GetFindingsOpts != nil {
+		allConfigs = cl.Spec.TableOptions.SecurityHubFindings.GetFindingsOpts
+		noTableConfig = false
+	}
+
 	svc := cl.Services().Securityhub
-	config := securityhub.GetFindingsInput{
-		MaxResults: 100,
+	var config securityhub.GetFindingsInput
+	getFindings := func(svc services.SecurityhubClient, config securityhub.GetFindingsInput) error {
+		p := securityhub.NewGetFindingsPaginator(svc, &config)
+		for p.HasMorePages() {
+			response, err := p.NextPage(ctx, func(o *securityhub.Options) {
+				o.Region = cl.Region
+			})
+			if err != nil {
+				return err
+			}
+			res <- response.Findings
+		}
+		return nil
```

Comment:
Once all of the configs are in a single array you can just iterate through it without having to have the unique case of no custom inputs

```suggestion
	for _, w := range allConfigs {
		config = w.GetFindingsInput
		p := securityhub.NewGetFindingsPaginator(svc, &config)
		for p.HasMorePages() {
			response, err := p.NextPage(ctx, func(o *securityhub.Options) {
				o.Region = cl.Region
			})
			if err != nil {
				return err
			}
			res <- response.Findings
		}
	} 
```
```